src/literals.py

```diff
@@ -4,7 +4,7 @@
 
 """Collection of global literals for the charm."""
 
-OPENSEARCH_DASHBOARDS_SNAP_REVISION = "22"
```

Comment:
On PR: https://github.com/canonical/opensearch-dashboards-operator/pull/109

You are jumping snap revisions from 19 > 22 (v2.15)

Here, you are rolling back 22 > 21 (v2.16?)

I would recommend, instead, to rerun the release process of OSD snap and get 2.16 -> rev. 23.